examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Owner.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<p>MAYBE: Remove commented code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>This is TODO</p>